docs/BR.md

```diff
@@ -853,30 +853,26 @@ The CA SHALL update and reissue CRLs at least (i) once every twelve months and (
 No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
-
-1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
-
-In the latter case, the OCSP signing Certificate MUST contain an extension of type id-pkix-ocsp-nocheck, as
-defined by RFC6960.
+The CA SHALL provide an OCSP responder that conforms to the profile specified by RFC5019 for all Certificates issued, including the transport profile.
 
 ### 4.9.10 On-line revocation checking requirements
-Effective 1 January 2013, the CA SHALL support an OCSP capability using the GET method for Certificates issued
-in accordance with these Requirements.
+All OCSP responses MUST conform to the profile specified in Section 7.3
 
-For the status of Subscriber Certificates:
+For newly issued Subscriber Certificates, OCSP responses SHOULD be published prior to the delivery of the Certificate to the Subscriber. OCSP responses MUST be published no later than 10 minutes after the Subscriber Certificate has been issued.
 
-The CA SHALL update information provided via an Online Certificate Status Protocol at least every four days. OCSP responses from this service MUST have a maximum expiration time of ten days.
+The CA SHALL publish an updated OCSP response for Subscriber Certificates by the earlier of:
+1. 8 hours prior to the nextUpdate field of the previously published OCSP response
+2. If the OCSP response has a validity period, as defined by the difference in time between the thisUpdate and nextUpdate fields, of greater than 16 hours, then the value of the thisUpdate field plus one half of the validity period of the OCSP response.
 
-For the status of Subordinate CA Certificates:
+The CA SHALL publish an updated OCSP response for Subordinate CA Certificates by the earlier of:
+1. 24 hours after revoking a Subordinate CA Certificate.
+2. 365 days
```

Comment:
@gerv There was ample discussion in the CA/Browser Forum on the ambiguity introduced by "every year" (is that calendar year? fiscal year? Does it happen to happen the same day the next year, or does 2017-12-31 count as 'every year' compared to 2016-01-01). So I think there was a general desire to be explicit on days.

You're absolutely correct that it cause a single day of drift every four years. This seems objectively better than causing a day of drift every three out of four years (366). I think we want to make sure we decisively paint this bikeshed so that we don't have to debate it on each individual proposal, and I hope you can agree that even 365 days - for a sub-CA - is almost certainly way too long.

The entire case is to support 'offline' signing, which involves work, but my desire here is to see something bounded at 30-60 days for the case of OCSP, because that's the worst-case that an attacker could continue to use a response. So this is baby steps towards a shorter period :)